src/CornerstoneViewport/CornerstoneViewport.js

```diff
@@ -83,6 +83,8 @@ class CornerstoneViewport extends Component {
     //
     style: PropTypes.object,
     className: PropTypes.string,
+    //isOverlayVisible conditionally rendering.    Initial commit    Initial commit
+    isOverlayVisible: PropTypes.bool.isRequired,
```

Comment:
Let's nix this comment and make this prop optional. Set default value to `true`

Reply:
Done